commands/service_list.go

```diff
@@ -30,21 +30,23 @@ To have more details, see the [detail command](mesg-core_service_detail.md).`,
 		Args:    cobra.NoArgs,
 		RunE:    c.runE,
 	})
+	c.cmd.Flags().BoolVarP(&c.quiet, "quiet", "q", c.quiet, "Display only the service hashes")
```

Comment:
I would change `quiet` to `--format` as this one is what we want.

Quiet is grate flag to be implemented globally and suppress all progress/messages that are not relevant for final output

Reply:
i will remove it from this PR, it needs more thoughts and it should have been two different PR anyway